plugin/storage/es/spanstore/reader.go

```diff
@@ -124,6 +127,8 @@ func (s *SpanReader) GetTrace(traceID model.TraceID) (*model.Trace, error) {
 func (s *SpanReader) readTrace(traceID string, traceQuery *spanstore.TraceQueryParameters) (*model.Trace, error) {
 	query := elastic.NewTermQuery(traceIDField, traceID)
 
+	// NB. Add an hour in both directions so that traces that straddle two indexes are retrieved.
+	// ie starts in one and ends in another.
```

Comment:
why "hour"? And why did we hardcode the granularity of index rotation to 24hrs in L196? It could've been configurable.